src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany

```

Comment:
Annotation Issue: The `@OneToMany` annotation should specify the `mappedBy` attribute to correctly define the relationship with the `Ticket` entity. Consider adding `cascade` and `fetch` attributes to manage the lifecycle and loading behavior of related entities.